internal/pkg/service/buffer/api/dependencies/dependencies.go

```diff
@@ -59,7 +59,7 @@ const (
 type ForServer interface {
 	serviceDependencies.ForService
 	BufferAPIHost() string
-	StatsAPINode() *statistics.APINode
+	StatsCollector() *statistics.CollectorNode
```

Comment:
Premenovane `statistics.APINode -> statistics.CollectorNode` aby bolo jasnejsie co to robi.